.github/workflows/sync-readme.yml

```diff
@@ -0,0 +1,23 @@
+name: Sync README.md to Docker Hub
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync:
+    name: Sync README
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout HEAD
+        uses: actions/checkout@v1
```

Comment:
I mean sure, `v1` will do the job, but why not use the latest available version?

Reply:
Yes.